content/src/roadmaps/industries/acupuncture.json

```diff
@@ -18,6 +18,12 @@
       "task": "determine-naics-code",
       "required": true
     },
+    {
+      "step": 1,
+      "weight": 100,
+      "task": "business-structure",
+      "required": true
+    },
```

Comment:
Was there a reason we never had a default add-on for tasks all industries shared?

Reply:
we actually used to and Dave removed it because there were a couple exceptions to it.... but I think it might be a good idea